python/ray/ml/examples/upload_to_wandb.py

```diff
@@ -0,0 +1,40 @@
+import ray
```

Comment:
can we add a top level comment/docstring explaining this? and maybe some extra inline comments, just to make it absolutely clear what we are doing (eg. what will be saved in wandb)